plugins/image-prioritizer/detect.js

```diff
@@ -51,10 +40,10 @@ function log( ...message ) {
  * @type {InitializeCallback}
  * @param {InitializeArgs} args Args.
  */
-export async function initialize( { isDebug, onLCP } ) {
+export async function initialize( { log, onLCP } ) {
```

Comment:
I just realized that if someone updated Embed Optimizer or Image Prioritizer before updating Optimization Detective, then they'll get JS errors. So we should add some logic to guard against this. For example:

```suggestion
export async function initialize( { log: _log, onLCP } ) {
	const log = _log || console.log; // TODO: Remove once Optimization Detective likely updated, or when strict version requirement added in od_init action.
```

The same should be done for Embed Optimizer.

Reply:
Done in 53b5f19.